src/datachain/cache.py

```diff
@@ -48,8 +48,6 @@ def get_parsed_location(self) -> Optional[dict]:
 
     def get_hash(self) -> str:
         fingerprint = f"{self.storage}/{self.path}/{self.version}/{self.etag}"
-        if self.location:
-            fingerprint += f"/{self.location}"
```

Comment:
Stopped using location in vfiles, but even if we keep it, I don't think it should be part of the hash since the file itself is the same even if there are rows that point to different locations in the file.